cmd/genkubedocs/postprocessing.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// MarkdownPostProcessing goes though the generated files
+func MarkdownPostProcessing(cmd *cobra.Command, dir string, processor func(string) string) error {
+	for _, c := range cmd.Commands() {
+		if !c.IsAvailableCommand() || c.IsHelpCommand() {
+			continue
+		}
+		if err := MarkdownPostProcessing(c, dir, processor); err != nil {
+			return err
+		}
+	}
+
+	basename := strings.Replace(cmd.CommandPath(), " ", "_", -1) + ".md"
+	filename := filepath.Join(dir, basename)
+
+	markdownBytes, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+
+	processedMarkDown := processor(string(markdownBytes))
+
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	_, err = file.WriteString(processedMarkDown)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// cleanupForInclude parts of markdown that will make difficult tu use it as include in the website:
+// - The title of the document (this allow more flexibility for include, e.g. include in tabs)
+// - The sections see also, that assumes file will be used as a main page
+func cleanupForInclude(md string) string {
+	lines := strings.Split(md, "\n")
+
+	cleanMd := ""
+	skipFirstLine := true
+	skipSeeAlso := false
+	for _, line := range lines {
+		if skipFirstLine {
+			skipFirstLine = false
+			continue
+		}
+		if line == "### SEE ALSO" {
```

Comment:
I'd do `if line == SEE ALSO { break }` so that this loop would just teminate directly, instead of continuing many times.